ArduCopter/Copter.h

```diff
@@ -895,6 +895,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     void do_aux_switch_function(int8_t ch_function, uint8_t ch_flag);
     void save_trim();
     void auto_trim();
+    bool allow_rc_override = true;
```

Comment:
I can see why putting allow-rc-overrides variable here makes some sense but it is inconsistent with the rest of Copter.h.  Normally variables and methods are defined in different places.  A better place would be within the ap_t structure or below the control_switch_state or perhaps way down below gndeffect_state,